sdk/core/Azure.Core/tests/DynamicData/MutableJsonElementTests.cs

```diff
@@ -294,6 +501,57 @@ internal static void ValidateToString(string json, MutableJsonElement element)
                 MutableJsonDocumentTests.RemoveWhiteSpace(element.ToString()));
         }
 
+        internal static string FormatDateTime(DateTime d)
+        {
+            byte[] bytes = JsonSerializer.SerializeToUtf8Bytes(d);
```

Comment:
Question here regarding whether we want to match JsonSerializer defaults, or have an opinion regarding the right format for Azure?